FrameworkSystem/scripts/dirac-admin-update-pilot.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+"""
+Script to update pilot version in CS
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+import DIRAC
+from DIRAC import S_OK, S_ERROR
+from DIRAC import gConfig, gLogger
+from DIRAC.Core.Base import Script
+
+Script.setUsageMessage('\n'.join([__doc__.split('\n')[1],
+                                  'Usage:',
+                                  '  %s version' % Script.scriptName,
+                                  ' ',
+                                  'Arguments:',
+                                  '  version:           pilot version you want to update to']
+                                 )
+                       )
+Script.registerSwitch(
+    "v", "vo=", "Location of pilot version in CS /Operations/<vo>/Pilot/Version"
+                " (default value specified in CS under /DIRAC/DefaultSetup)")
+
+Script.parseCommandLine(ignoreErrors=False)
+
+args = Script.getPositionalArgs()
+if len(args) < 1 or len(args) > 2:
+  Script.showHelp()
+
+version = args[0]
+vo = None
+for switch in Script.getUnprocessedSwitches():
+  if switch[0].lower() == "v" or switch[0].lower() == "vo":
```

Comment:
I still think that one should not give the impression that flags can be case insensitive.